config/cookstyle.yml

```diff
@@ -105,6 +105,12 @@ Chef/WhyRunSupportedTrue:
   Description: why_run_supported? no longer needs to be set to true as it is the default in Chef 13+
   Enabled: true
 
+PropertyWithNameAttribute:
+    Description: Resource property sets name_attribute not name_property
+    Enabled: true
+    #Include:
+    #  - '**/resources/*'
```

Comment:
we'll want to enable this at some point, but I'm not ready for it yet

Reply:
(can't imagine people put non-rb files in the resources dir, but they certain want to do that in the attributes directory so its probably a good habit to get into)

Reply:
Updated